_posts/2016-10-23-recurse-center-week-4.md

```diff
@@ -0,0 +1,158 @@
+Whenever I put my mind in NVC mode, it usually results in me overanalyzing everything. Framing things in terms of needs/feelings over and over again:
+
+> Ugh! I feel X because of Y!
+> ...no I feel X because of unmet needs.  
+> ...wait X is a strategy, not a feeling.  
+> Ugh! I suck at NVC!  
+> ...that's a moralistic judgement about my abilities with NVC!  
+> ...shit.  
+
+The result is that I'm quickly sucked into the idea that there are right and wrong ways to think. Thinking about thoughts until the thoughts are the things I want to think.
+
+It took me Monday and Tuesday to get the presentation where I wanted. After that I still hadn't blogged. I switched from one heavily self-reflective task to the next. By the end of it all I just wanted to code!
+
+#### **Misusing NVC**
+
+When I posted that I would be talking about NVC, it grabbed a fellow Recurser's attention. It was nice having someone to run ideas by. Also to have that person point out where I had been misusing NVC.
+
+In the past I've gotten caught up in the game of *"How can we translate everything to be NVC"*. That would then quickly lead me to judging all my communication as good/bad. Seeing NVC as a tool rather than "THE" way to communicate helped relieve some pressure from myself. There's no right answers!
+
+### Social 👫
+
+#### **Getting sucked back into RC.**
+
+This week had a bunch of awesome events at RC.
+
+- Contra dancing  
+- Non-technical talks  
+- Graph Theory Workshop  
+- Spelling Bee Coding Game  
+- Job Fair  
+
+It's hard finding a balance with these events.  
+Three RC events in a row meant that Monday, Tuesday, Wednesday I spent my whole day at RC.
+
+(Thursday I went to BrooklynJS, but would have been at RC if that hadn't been the case.)
+
+All this time at RC really anchors my sense of friendship and community to one space: RC. And one purpose: coding.
+
+#### **Having spaces outside RC.**
+
+Thankfully I've been feeling better about my relationships outside of RC.
+
+Saturday some friends came out to an Electro Swing event with me. It felt SO good to get out dancing again.
+
+Sunday I was able to have a group cooking/potluck sesh. Group dinners are one of my favorite things. Such a wonderful sense of community that comes from cooking/sharing food together.
+
+Both these things were with people from RC and that just made it all the better 😍.
+
+### Intellectual 🔭
+
+#### **Asynchrnous `nodepub`**
+
+After a few days of doing nothing but processing thoughts, I was SO ready to crank out some code.
+
+I disconnected, went into a room and started working on a PR for `nodepub`. It's an open source library for bundling HTML into `epub` files.
+
+The library has been working well for me, but unfortunately was using a some synchronous apis. In the NodeJS world, this means the entire server can get blocked on writing epub files. I thought making the code fully asynchronous would be a fun little project. The last two days of the week I did that and opened a PR.
```

Comment:
Typo: remove `a` from `using a some asynchronous apis`. Also consider changing `apis` to `APIs`.